src/PanelTraits/Update.php

```diff
@@ -73,4 +75,31 @@ public function getUpdateFields($id)
 
         return $fields;
     }
+
+    /**
+     * Get the value of the 'name' attribute from the declared relation model in the given field.
+     *
+     * @param Model $model The current CRUD model.
+     * @param array $field The CRUD field array.
+     *
+     * @return mixed The value of the 'name' attribute from the relation model.
```

Comment:
Why is that a mixed?

Reply:
It returns an attribute of a model. It can be a string, int, date, etc.

Reply:
Ah, ok.